src/parsers/psse_dynamic_data.jl

```diff
@@ -61,10 +61,15 @@ function _parse_input_types(_v, val)
         return NaN
         # If the parameter is a tuple (as a string), then construct the tuple directly.
     elseif isa(_v, String)
+        #TODO: Generalize n-length tuple
         m = match(r"^\((\d+)\s*,\s*(\d+)\)$", _v)
+        m2 = match(r"^\((\d+)\s*,\s*(\d+),\s*(\d+)\)$", _v)
```

Comment:
@jd-lara Any idea how to generalize this Regex for a tuple of n-length? The PIDGOV model and WPIDHY models have a tuple with three terms.

Reply:
Let's do this for another time. For now this should be fine